bootstrap/kubeadm/types/upstreamv1beta3/conversion_test.go

```diff
@@ -107,3 +110,25 @@ func joinControlPlanesFuzzer(obj *JoinControlPlane, c fuzz.Continue) {
 
 	obj.CertificateKey = ""
 }
+
+// Custom fuzzers for CABPK v1beta1 types.
+// NOTES:
+// - When fields do not exist in kubeadm v1beta4 types, pinning them to avoid cabpk v1beta1 --> kubeadm v1beta4 --> cabpk v1beta1 round trip errors.
```

Comment:
```suggestion
// - When fields do not exist in kubeadm v1beta4 types, pinning them to avoid cabpk v1beta1 --> kubeadm v1beta3 --> cabpk v1beta1 round trip errors.
```
nit